src/test/kotlin/org/taktik/icure/testutils/http.kt

```diff
@@ -0,0 +1,38 @@
+package org.taktik.icure.testutils
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.matchers.shouldBe
+import org.springframework.http.HttpStatus
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import org.springframework.web.util.UriUtils
+
+val authorizationString by lazy {
+	"Basic ${
+		java.util.Base64.getEncoder()
+			.encodeToString("${System.getenv("ICURE_COUCHDB_TEST_USER")}:${System.getenv("ICURE_COUCHDB_TEST_PWD")}".toByteArray())
```

Comment:
There is already the env vars ICURE_COUCHDB_USERNAME & ICURE_COUCHDB_PASSWORD in ICureBackendApplication which are used : Let's align to those ones

Reply:
The name is actually wrong, the variables are supposed to hold the username and password of the icure test user, not of the couchdb admin. I renamed the variables to `ICURE_TEST_USER_NAME` and `ICURE_TEST_USER_PASSWORD`  to better represent the intention.
Note that when merged will also break existing test configurations, as the variables were used also in other existing tests.